spec/decorators/content_items_decorator_spec.rb

```diff
@@ -18,31 +18,28 @@
 
   context "with organisation filter" do
     it "renders the organisation title as page title" do
-      allow(helpers).to receive(:params).and_return(organisation_slug: "the-slug")
       content_items = [build(:content_item, organisations: [organisation])]
```

Comment:
None of the tests need `content_items` anymore, the decorator can be instantiated with an empty collection and still pass the tests e.g. `subject = ContentItemsDecorator.new([])`